### PR TITLE
Made the tests ensure that Cluster Autoscaler is on before running.

### PR DESCRIPTION
**Pull request by wasylkowski**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: test/e2e/autoscaling/cluster_autoscaler_scalability.go [hunk 1]
```diff
@@ -69,6 +69,12 @@ var _ = framework.KubeDescribe("Cluster size autoscaler scalability [Slow]", fun
 	BeforeEach(func() {
 		framework.SkipUnlessProviderIs("gce", "gke", "kubemark")
 
+		// Check if Cloud Autoscaler is enabled by trying to get its ConfigMap.
+		_, err := f.ClientSet.CoreV1().ConfigMaps("kube-system").Get("cluster-autoscaler-status", metav1.GetOptions{})
```
Comment (mwielgus) [inline test/e2e/autoscaling/cluster_autoscaler_scalability.go]: What about retries?

Comment (wasylkowski) [reply test/e2e/autoscaling/cluster_autoscaler_scalability.go]: What do you mean by that? Do you mean that this code should retry checks?

Comment (mwielgus) [reply test/e2e/autoscaling/cluster_autoscaler_scalability.go]: Yes, for, lets say 30 sec.

Comment (wasylkowski) [reply test/e2e/autoscaling/cluster_autoscaler_scalability.go]: But why? What can happen that will make this fail for 20 seconds and then succeed? AFAIU CA is either on or not. Note that I am not questioning the general idea that if something can fail transiently we should retry, but for the moment I don't see how can any failure here be transient.

Comment (mwielgus) [reply test/e2e/autoscaling/cluster_autoscaler_scalability.go]: Talked f2f. @wasylkowski explained that the problem that he was trying was not that CA started too slow but that the potential developer might forget to turn CA on and the test would run for long time before signalising the problems.

**Pull request merged**